### DISCUSSION TITLE
Nemo - thumbnails on folders

### DISCUSSION THREAD

**Discussion by HT-7** [linuxmint/discussions#116] [Ideas]
Since Windows XP, Windows Explorer can display preview thumbnails on folders from files inside the folders. It would be convenient to have this option in Nemo too.

On solid state drives, it barely makes a performance difference, but adds convenience.

Obviously, for the Media Transfer Protocol (MTP), having this would be impractical because MTP is enormously slow and laggy by design. Windows Explorer does not do it on MTP either. But on solid state devices, it makes no major performance difference while adding convenience.

Perhaps a limiting option can be made, such as "display folder thumbnails for the first 20 listed folders", to protect against lags from highly populated folders. The user will adjust it based on performance.

**Comment by Secret-chest** [comment]
It wouldn't work for all icon themes.
